src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.xml

```diff
@@ -34,6 +34,10 @@
             <tag name="texter.transport_factory" />
         </service>
 
+        <service id="notifier.transport_factory.infobip" class="Symfony\Component\Notifier\Bridge\Infobip\InfobipTransportFactory" parent="notifier.transport_factory.abstract">
+            <tag name="texter.transport_factory" />
+        </service>
```

Comment:
You should rebase on current master and move this to the new transport.php file.

Reply:
Done!